### PR TITLE
Add FieldSetter filter

### PR DESCRIPTION
**Pull request by phanimarupaka**
@pwittrock 

This PR is to add Fieldsetter filter.

### CODE REVIEW

File: kyaml/setters2/settersutil/fieldsetter.go [hunk 1]
```diff
@@ -20,6 +21,17 @@ type FieldSetter struct {
 	Description string
 
 	SetBy string
+
+	Count int
+
+	OpenapiPath string
```
Comment (pwittrock) [inline kyaml/setters2/settersutil/fieldsetter.go]: OpenAPIPath

Comment (phanimarupaka) [reply kyaml/setters2/settersutil/fieldsetter.go]: Done.

**Pull request merged**